src/microbe_stage/OrganelleDefinition.cs

```diff
@@ -168,6 +168,11 @@ microbe editor.
     /// </summary>
     public bool Unique;
 
+    /// <summary>
+    ///   Determines whether this organelle appears in LAWK-only games
+    /// </summary>
+    public bool Lawk;
```

Comment:
I think naming this to LAWK would make more sense as it is not a normal word (need to also add to the jetbrains dotsettings file so that it doesn't say this is mispelled).